src/formats/apple-notes/sqlite/index.js

```diff
@@ -1,12 +1,14 @@
 import { Platform } from 'obsidian';
 import { os, path } from '../../../filesystem';
+import { supportsJson, sqlToJson } from './fallback';
 import { error, raw, sql } from './utils.js';
 
-const crypto = Platform.isDesktopApp ? window.require('node:crypto') : null;
-const child_process = Platform.isDesktopApp ? window.require('node:child_process') : null;
+export const crypto = Platform.isDesktopApp ? window.require('node:crypto') : null;
```

Comment:
Why do these need to be exported?

Reply:
At least one is used in the new ‘fallback.js’ file.